### PR TITLE
feat : added wish-list page

### PR DESCRIPTION
**Pull request by ArjunJunna**
added wish-list page

### CODE REVIEW

File: wish-list.html [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <title>Petmate</title>
+  </head>
+  <body>
+    <div class="page-container">
+      <header class="header">
+        <div class="header__wrapper">
```
Comment (jalajcodes) [inline wish-list.html:21]: You can change div to nav to make code more semantic.

**Review by jalajcodes — APPROVED** [review]
Looks good. You can merge it

File: css/pages/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.dismiss-v{
+  position: absolute;
```
Comment (anik31) [inline css/pages/wishlist.css]: suggestion - add more descriptive class name as dismiss-v doesn't give a context

File: main.css [hunk 3]
```diff
@@ -14,3 +14,4 @@
 @import '/css/pages/auth.css';
 @import '/css/pages/landingPage.css';
 @import '/css/pages/cart-page.css';
```
Comment (anik31) [inline main.css]: suggestion - maintain a consistent file naming. In landing page you have used camelcase & in cart page you have used hyphen. Anything works fine but I will suggest using a single method for file naming.

File: wish-list.html [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+      <main>
+        
+        <div class="hero-heading">Your Wishlist</div>
+        <div class="products-5">
+          <article class="component">
+            <div class="card__container">
+                 
+              <div class="card-header">
+                <i class="bi bi-x-circle dismiss-v"></i>
+                <img src="/images/drools-image.png" alt="product image" class="card-image" />
```
Comment (anik31) [inline wish-list.html]: suggestion - for alt you can add about the contents of the image.

**Review by anik31 — COMMENTED** [review]
@ArjunJunna you have written good code & the UI also looks good 😻. I have added some suggestions, you can check them and work on them later if you have time constraints.
Please include the deployed link or some screenshots of the UI in the PR description.

**Review by Dev-AkshatGupta — APPROVED** [review]
Looks cool to me and you UI is also applauding

File: wish-list.html (hunk 1 above)
Comment (ArjunJunna) [reply wish-list.html:21]: sure. Thank you.

File: css/pages/wishlist.css (hunk 2 above)
Comment (ArjunJunna) [reply css/pages/wishlist.css]: sure will make the changes. Thank you for appreciating the UI.

File: main.css (hunk 3 above)
Comment (ArjunJunna) [reply main.css]: sure will make the changes.

**Comment by ArjunJunna** [comment]
> Looks cool to me and you UI is also applauding

Thanks a lot Akshat

**Pull request merged**